scripts/packaging/sync_controller_lib.sh

```diff
@@ -40,12 +40,17 @@ else
     git checkout -b ${VERSION}
 fi
 
-# Copy headers
+# Copy headers and C++ source code
 if [ "${OSTYPE}" != "msys" ]; then
     # don't copy the include files on Windows as they are the same as on other platforms, and they generate a huge diff due to Windows line endings which differ from Linux/macOS.
     rm -rf include
     mkdir -p include
     cp -r ${WEBOTS_HOME}/include/controller/* include
+    cp include/plugins/robot_window/{robot_window.h,robot_wwi.h} include
```

Comment:
Is this path correct?
Because the files are located in ``${WEBOTS_HOME}/include/controller/c/webots/plugins/robot_window``, so it seems like the `c/webots/` part is missing.

Reply:
You're right, but I am surprised I haven't got any errors while testing. It seems I manually put the file there while testing